add-hotel.php

```diff
@@ -10,119 +13,129 @@
 </head>
 
 <body>
-    <div class="container-fluid">
-        <div class="row">
-            <div class="col-12 col-md-8 col-lg-8 offset-md-2 offset-lg-2">
-                <div class="row">
-                    <div class="col-12">
-                        <center>
-                            <h4 class="stf-sub-heading">Add New Hotel</h4>
-                        </center>
-                    </div>
-                    <div class="col-12">
-                        <hr>
-                    </div>
-                    <div class="col-12 mt-3">
-                        <div class="row">
-                            <div class="col-12 col-md-4 col-lg-4">
-                                <label class="descriptions">Hotel Name</label>
-                            </div>
-                            <div class="col-12 col-md-8 col-lg-8">
-                                <input type="text" class="w-100" id="h-name">
-                            </div>
+    <?php
+    if (isset($_SESSION["user"])) {
```

Comment:
Web content is only visible when the `user session object` is link with user. however, if the user session is lost, a **blank web page** may be rendered. So, include the `else` code fragment and redirect to the proper page when the user session object is missing.